File: beecrowd_urionlinejudge/uri1009SalarioBonus.c
#include <stdio.h>

int main() {
char NomeFuncionario[200];
double SalarioFixo,Vendas;

scanf("%s", NomeFuncionario);
scanf("%lf", &SalarioFixo);
scanf("%lf", &Vendas);


printf("TOTAL = R$ %0.2lf\n",SalarioFixo+(Vendas*0.15));
    
	return 0;
}

File: beecrowd_urionlinejudge/uri1009SalarioBonus.pas
program main;
var NomeFuncionario : string;
var SalarioFixo,Vendas : real ;

Begin

   read(NomeFuncionario);
   read(SalarioFixo);
   read(Vendas);
   
   
writeln('TOTAL = R$ ',(SalarioFixo+(Vendas*real(0.15))):1:2);

end.
